### PR TITLE
Open source the deployment

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+2. in [vars](vars/) copy [vars/template.yml](vars/template.yml) to `{deployment}.yml` where `{deployment}` is one of `prod` or `stg` and fill in values
+3. `dnf install ansible origin-clients python3-openshift`
+4. `DEPLOYMENT={deployment} make deploy`
+
+## What's in here
+
+- [playbooks](playbooks/) - Ansible playbooks.
+- [vars](vars/) - Variable file(s). See section below for more info.
+- [Openshift](openshift/) - Openshift resource configuration files (templates).
+- [secrets](secrets/) - secret stuff to be used from `openshift/secret-*.yml.j2`
```
Comment (jpopelka) [inline README.md]:
```suggestion
```

File: openshift/deployment-worker.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          secretName: packit-config
+      - name: packit-vol-1
+        persistentVolumeClaim:
+          claimName: packit-worker-pvc-1
+      - name: packit-vol-2
+        persistentVolumeClaim:
+          claimName: packit-worker-pvc-2
+      containers:
+      - name: packit-worker-1
+        image: docker.io/rpitonak/packit-service:master
```
Comment (jpopelka) [inline openshift/deployment-worker.yml]: Is this something temporary or why is that in your namespace?

File: README.md (hunk 1 above)
Comment (rpitonak) [reply README.md]: secrets dir is still there, but now just with README and template.

File: openshift/deployment-worker.yml (hunk 2 above)
Comment (rpitonak) [reply openshift/deployment-worker.yml]: Good catch, this is a leftover. I will fix it.

**Comment by rpitonak** [comment]
Thank you for the review I tested it locally and it works well. I will merge this. Feel free to open an issue if you will find anything.

**Pull request merged**